visualisation/meshViewer.cpp

```diff
@@ -110,6 +110,45 @@ using namespace DGtal;
  */
 
 
+/**
+ * Custom Viewer3D to override KeyPressEvent method and handle new key display.
+ * It also desactivate the double Rendering mode for more efficiency.
+ **/
+template<typename Space = SpaceND<3>, typename KSpace = KhalimskySpaceND<3> >
+class CustomViewer3D: public Viewer3D<Space, KSpace>
```

Comment:
1) It was just to move this subclass in 3dSDPViewer.h, not to add a specific .h
2) It will works because you don't use Space and KSpace in your code, you just need to replace all the "Viewer3D<Space, KSpace>" by "Viewer3D<>". Templated a local class only used with the default parameters seems to me useless.